uqcsbot/api.py

```diff
@@ -24,3 +26,83 @@ def get_members(self) -> List[str]:
             members += page["members"]
         return members
 
+
+class APIMethodProxy(object):
+    """
+    Helper class used to implement APIWrapper
+    """
+    def __init__(self, client: SlackClient, method: str, is_async: bool = False):
+        self._client = client
+        self._method = method
+        self._async = is_async
+
+    def __call__(self, timeout=None, **kwargs) -> Union[dict, Awaitable[dict]]:
```

Comment:
Can you add documentation for the timeout parameter? What happens when it times out?

Reply:
The timeout parameter is just to replicate the ability of `SlackClient.api_call` to pass in timeout as a positional argument. I could remove `timeout=None,` and this would function effectively the same. Just gets passed straight through to `SlackClient.api_call`

Happy to still document it if you want

Reply:
Ah ok, if you can that'd be good but I'm not too fussed about it if it's just that.

Reply:
Just changed it to `**kwargs` instead